### PR TITLE
FRW-42 DevVM Deprecation

### PR DESCRIPTION
**Pull request by geega**
Ticket: https://spryker.atlassian.net/browse/FRW-42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Spryker OS - Reference repository for DevVM
 
+**DEPRECATED**
```
Comment (asmarovydlo) [inline README.md:3]: Please a message that docker should be used instead, and the link to the documentation.

**Review by asmarovydlo — CHANGES_REQUESTED** [review]
The default branch for the repository is not master, please provide the PR there.

**Comment by asmarovydlo** [comment]
https://github.com/spryker/devvm/pull/94#pullrequestreview-1035497585 
Currently, the PR have conflicts, because those branches have different readme file.
Please create 2 PR's (keep this one for `master` branch), and create another one for `develop`.

**Comment by geega** [comment]
Created https://github.com/spryker/devvm/pull/95

**Review by asmarovydlo — APPROVED** [review]

**Pull request closed without merging**